src/Snak/SnakObject.php

```diff
@@ -97,12 +97,12 @@ public function equals( $target ) {
 	/**
 	 * @see Serializable::serialize
 	 *
-	 * @since 0.1
+	 * @since 7.0
```

Comment:
I believe the returned value is a crucial part of the contract of a method. If the format of the return value changes, it's a different method. The idea of the since tag is to make this obvious.